File: Modul_4/Bahasa Python/Prak401_2310817210013_MuhammadAufaFitrianda.py
kelipatan = int(input("Masukkan bilangan kelipatan: "))
simbol = input("Masukkan simbol yang akan digunakan: ")

for i in range(1, 51):
    if i % kelipatan == 0:
        print(simbol, end=' ')
    else:
        print(i, end=' ')

File: Tugas_Praktikum_Pemograman_1_Muhammad_Aufa_Fitrianda_2310817210013/Modul_4/Bahasa_C/Prak401_2310817210013_MuhammadAufaFitrianda.c
#include <stdio.h>

int main() {
    int kelipatan;
    char simbol;
    
    printf("Masukkan Bilangan kelipatan Anda\t: ");
    scanf("%d", &kelipatan);
    
    printf("Masukkan Simbol Anda\t\t\t: ");
    scanf(" %c", &simbol);

    for (int i = 1; i <= 50; i++) {
        if (i % kelipatan == 0) {
            printf("%c ", simbol);
        } else {
            printf("%d ", i);
        }
    }

    return 0;
}
